app/components/app/app.component.ts

```diff
@@ -24,7 +24,7 @@ export class AppComponent {
     // If user has previously saved a color, then set the app to that color
     if (fs.existsSync(userColorFilePath)) {
       fs.readFile(userColorFilePath, function(err, buffer) {
-        console.log(buffer.toString());
+        console.log("Last used theme: " + buffer.toString() + ", initializing to this theme.");
```

Comment:
I wonder if using the console log to notify theme change is really necessary? I feel like the console is a little crowded, but that is subjective. Let me know what you think...